Libraries/Components/View/View.js

```diff
@@ -256,6 +256,38 @@ const View = React.createClass({
     onMagicTap: PropTypes.func,
 
     /**
+     * *(Apple TV only)* Optional method.  When implemented, this view will be focusable
+     * and navigable using the Apple TV remote.
+     */
+    onTVSelect: PropTypes.func,
+
+    /**
+     * *(Apple TV only)* Optional method. Will be called if this view comes into focus
+     * during navigation with the TV remote.  May be used to give the view a different
+     * appearance when focused.
+     */
+    onTVFocus: PropTypes.func,
+
+    /**
+     * *(Apple TV only)* Optional method.  Will be called if this view leaves focus during
+     * navigation with the TV remote.
+     */
+    onTVBlur: PropTypes.func,
+
+    /**
+     * *(Apple TV only)* Optional method.  When implemented, this method will be called when
+     * the user presses a button or makes a swipe gesture on the TV remote.  The event passed
+     * into this method will have a nativeEvent property, with a type that is one of
+     * [left, right, up, down, playPause, menu].
+     */
+    onTVNavEvent: PropTypes.func,
+
+    /**
+     * *(Apple TV only)* May be set to true to force the Apple TV focus engine to move focus to this view.
+     */
+    hasTVPreferredFocus: PropTypes.bool,
```

Comment:
I think I'd much rather have us namespace all of these under a separate component rather than adding platform keyed properties to View.